nvme.h

```diff
@@ -90,6 +91,7 @@ struct nvme_config {
 		OPT_FMT("output-format", 'o', &nvme_cfg.output_format, output_format), \
 		##__VA_ARGS__,                                                         \
 		OPT_UINT("timeout",      't', &nvme_cfg.timeout,       timeout),       \
+		OPT_FLAG("dry-run",      'd', &nvme_cfg.dry_run,       dry_run),       \
```

Comment:
Let's use only the long option name for it. It's likely one command is already using `-d` and I'd like to avoid further confusion with the short options name.